intellij-plugin/src/main/resources/messages/ApolloBundle.properties

```
@@ -121,5 +121,9 @@ inspection.unusedField.quickFix=Delete field
 inspection.apollo4Available.displayName=Apollo Kotlin 4 is available
 inspection.apollo4Available.reportText=Apollo Kotlin 4 is available
 inspection.apollo4Available.quickFix=Migrate to Apollo Kotlin 4
+inspection.missingIntrospection.displayName=No introspection or registry block
+inspection.missingIntrospection.reportText=No introspection or registry block
+inspection.missingIntrospection.quickFix.introspection=Add introspection block
+inspection.missingIntrospection.quickFix.registry=Add registry block
```

I'd vote for skipping over registry for now and focus on getting the endpoint configured. As said above, I'd say this is the biggest win. It just happens to be in an `introspection {}` block but we might want to evolve the API to just `endoint` or so.

```suggestion
inspection.missingIntrospection.displayName=GraphQL endpoint not configured
inspection.missingIntrospection.reportText=GraphQL endpoint not configured
inspection.missingIntrospection.quickFix.introspection=Add introspection block
```

---

Makes sense, agreed!